app/views/showApplication.scala.html

```diff
@@ -218,7 +216,7 @@ <h2 class="mdl-card__title-text">@application.subject</h2>
             <div class="mdl-list mdl-cell mdl-cell--12-col">
                 Créée par
                 <div class="mdl-list__item">
-                    <span class="mdl-list__item-primary-content">@Map("Aide A+" -> "aplus", "Pôle Emploi" -> "pe", "CPAM" -> "cpam", "CAF" -> "caf", "CNAV" -> "cnav", "DGFIP" -> "dgfip", "Défenseur des droits" -> "ddd").find(a => application.creatorUserName.contains(a._1)) match {
+                    <span class="mdl-list__item-primary-content">@Map("A+" -> "aplus", "Pôle Emploi".toUpperCase() -> "pe", "MSA" -> "msa", "CPAM" -> "cpam", "CRAM" -> "cpam", "CPAM" -> "cpam", "CNAM" -> "cpam", "CAF" -> "caf", "CNAV" -> "cnav", "CARSAT" -> "cnav", "DGFIP" -> "dgfip", "DDFIP" -> "dgfip", "DRFIP" -> "dgfip",  "Défenseur des droits".toUpperCase() -> "ddd").find(a => application.creatorUserName.toUpperCase.contains(a._1)) match {
```

Comment:
tu as dupliqué "CPAM" -> "cpam"
```suggestion
                    <span class="mdl-list__item-primary-content">@Map("A+" -> "aplus", "Pôle Emploi".toUpperCase() -> "pe", "MSA" -> "msa", "CPAM" -> "cpam", "CRAM" -> "cpam", "CNAM" -> "cpam", "CAF" -> "caf", "CNAV" -> "cnav", "CARSAT" -> "cnav", "DGFIP" -> "dgfip", "DDFIP" -> "dgfip", "DRFIP" -> "dgfip",  "Défenseur des droits".toUpperCase() -> "ddd").find(a => application.creatorUserName.toUpperCase.contains(a._1)) match {
```